test.py

```diff
@@ -1,18 +1,18 @@
-from EthLeverageWallet import MyContract
+from src.EthLeverageWallet import EthLeverage
 from web3 import Web3
 
-http_rpc = None # https://mainnet.infura.io/v3/...
+http_rpc = None  # https://mainnet.infura.io/v3/...
 
 # Automatically takes `w3.eth.accounts[0]` as the main account
-myContract = MyContract().init(http_rpc)
-myContract.buildContract()
+i_EthLW = EthLeverage().init(http_rpc)
```

Comment:
I used the name EthLeverage  because it's more meaningful than MyContract. i_EthLW stands for "instance of EthLeverageWallet"